app/assets/stylesheets/_misc_legacy.sass

```diff
@@ -166,15 +166,10 @@ div
       margin-bottom: 1em
       td.checkbox
         display: none
-    .wiki
-      h1
-        &:first-child
-          display: none
-        font-size: 120%
-      h2
-        font-size: 110%
-    .generic-table--no-results-container
-      max-width: calc(100% - #{$version-summary-width})
+
+  .generic-table--no-results-container
```

Comment:
@silkentrance The changes look good! I only have one little remark left. This rule should be of the same indentation as before. So it should only be applied in the scope of `#roadmap`.